src/pages/[language]/index.vue

```diff
@@ -353,7 +349,7 @@
       grid-column-end: 21;
     }
 
-    .page-index__blog-posts-list-item {
+    .page-index__blog-posts-list li {
```

Comment:
What do you think of this page styling? Ideally we don't have any page styling that targets component classes.

Reply:
Ok, but I don't know of a good alternative, and the custom styling is necessary to have it looking like it does now on the homepage.

Reply:
I had to add a `div` wrapper to get the same grid behaviour without using the `li`, what do you think?

Reply:
Yes nice!